tests/Keboola/GoogleAnalyticsExtractor/ApplicationTest.php

```diff
@@ -154,50 +144,6 @@ public function testAppRunProperties(): void
         Assert::assertEquals(1, count($usersManifests));
     }
 
-    public function testAppUserException(): void
```

Comment:
testy pro odstraněný Universal Analytics, pomalu vypínají služby tak musíme odmazávat testy...chtělo by to teda pročistit i kod